README.md

```diff
@@ -23,19 +23,27 @@ English | [简体中文](./README.zh-CN.md)
 
 Dashboard for [Apache APISIX](https://github.com/apache/apisix)
 
+The goal of the Apache APISIX Dashboard project is to enable everyone to quickly experience and learn Apache APISIX, and it cannot be used directly in a production environment. Therefore, its function point coverage is always a subset of Apache APISIX, and may lag behind the rapid iteration of Apache APISIX. If you need to use the Dashboard project in a production system, you need to enhance user permissions, communication security, high availability and advanced features.
+
 ## User Guide
 
 Please refer to [User Guide](./docs/USER_GUIDE.md)
 
-## Deployment
+## For Developer
+
+If you are a developer, please refer to start manager-api and Web separately
+
+- [dependencies](#dependencies)
```

Comment:
1. Capitalize the first letter
2. Can we write the description directly here? then we can remove `## dependencies`

eg:

- [Dependencies](#dependencies): apisix-dashboard provides a management interface for [Apache APISIX](https://github.com/apache/apisix), we need to install [APISIX](https://github.com/apache/apisix#configure-and-installation) first.
- [Develop dashboard manual](./docs/develop.md).